.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "cSpell.ignoreWords": [
+        "alt",
+        "flix",
+        "preconnect"
+    ]
+}
```

Comment:
Even though it's a config file for a specific IDE (VSCode), it's [a good practice to add a single empty line at the end of a source file](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=If%20you%20try%20to%20concatenate,ends%20with%20a%20newline%20character.&text=The%20empty%20line%20in%20the,you%20have%20reached%20the%20end.) to ensure a consistent behavior on different UNIX-like tools. This [could be enforced via ESLint](https://eslint.org/docs/rules/eol-last).

Reply:
@FelipePavan13 
PS: the VSCode settings file usually is user preference thing. If it's not a project convention (as in "every other dev here must use VSCode"), this should be added to the `.gitignore` file.

As the project does not have any Readme so far, I could not check for project conventions/practices

Reply:
thanks for the input!